concepticondata/concepticon.tsv

```diff
@@ -2677,3 +2677,7 @@ ID	GLOSS	SEMANTICFIELD	DEFINITION	ONTOLOGICAL_CATEGORY	REPLACEMENT_ID
 2676	TWO HUNDRED	Quantity	The cardinal number occurring after one hundred and ninety-nine and before two hundred one, represented in Roman numerals as CC and in Arabic numerals as 200.	Number	
 2677	SPEARTHROWER	Warfare and hunting	A spear-thrower or atlatl is a tool that uses leverage to achieve greater velocity in dart-throwing, and includes a bearing surface which allows the user to store energy during the throw.	Person/Thing	
 2678	HAIR OR FUR	The body	The cover of a mammal or a bird.	Person/Thing	
+2679	BLOW (WITH NOSE)	Basic actions and technology	To expel air through nose.	Action/Process	
+2670	MACHETE	Food and drink	A machete is a broad blade used either as an implement like an axe, or in combat like a short sword.	Person/Thing	
```

Comment:
ah, this is the problem: you started re-counting from 2679 to 2670, but need to count from 2680, etc. I had the same problem in the past ;-)